### DISCUSSION TITLE
explosion-effect config description inconsistency

### DISCUSSION THREAD

**Discussion by Blackrobe** [PurpurMC/Purpur#1338] [__Dismissed__] [answered]
Hi, I want to report inconsistency in the documentation vs actual server value
https://purpurmc.org/docs/Configuration/explosion-effect_2

Key: `end-crystal.*.explosion-effect` (and possibly others like respawn anchor or bed in the nether)

It is said in the documentation
> explosion-effect
>     default: DESTROY
>     description: The type of explosion effect. Available values are NONE, BREAK, and DESTROY.

But when I try to set it to "BREAK" it is not accepted by the server:
```
[09:05:13 ERROR]: Unknown value for blocks.end-crystal.baseless.explosion-effect! Using default of BLOCK
[09:05:13 ERROR]: Unknown value for blocks.end-crystal.base.explosion-effect! Using default of BLOCK
```

Also in the documentation, the "BLOCK" option is nowhere to be seen in any config.

**Comment by granny** [comment]
please move this to an issue on https://github.com/PurpurMC/PurpurDocs

**Reply by BillyGalbreath** [reply, marked as answer]
This should actually be a bug report on Purpur itself. It looks like somewhere along the lines we started using `Level#ExplosionInteraction` instead of `Explosion#BlockInteraction`

**Reply by granny** [reply]
Ah, I see.

**Reply by Blackrobe** [reply]
Thank you @granny , will move to PurpurDocs https://github.com/PurpurMC/PurpurDocs/issues/72.

I neither know how to code in Java nor familiar with Purpur's codebase though, so I don't really know how to file an accurate bug report on this one.
